packages/api/src/controllers/sensorController.js

```diff
@@ -231,6 +231,31 @@ const sensorController = {
     }
   },
 
+  async getReadingsByFarmId(req, res) {
+    try {
+      const { farm_id, days } = req.params;
+      if (!farm_id) {
+        return res.status(400).send('Invalid farm id');
+      }
+      let result;
+      if (!days) {
+        result = await SensorReadingModel.query()
```

Comment:
Please keep all knex and Objection API calls in the model classes. See [db-models](https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1191575576/Pre-integration+peer+review#Backend-code)

Reply:
Moved them to model classes!